pkg/kubecfg/pack.go

```diff
@@ -23,12 +23,16 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
+	"log"
```

Comment:
for consistency, let's use the same logging framework as the rest of the codebase:

```suggestion
	log "github.com/sirupsen/logrus"
```